### PR TITLE
Fixes #5421: Add the right command to make Jetty run on all interfaces

### PR DESCRIPTION
**Pull request by Kegeruneku**
Ticket: http://www.rudder-project.org/redmine/issues/5421

To be merged in master

### CODE REVIEW

File: scripts/rudder-multiserver-setup/rudder-ldap.sh [hunk 1]
```diff
@@ -22,7 +22,8 @@ sed -i "s/^IP=.*$/IP=*/" /etc/default/rudder-slapd
 service rudder-slapd restart
 
 # Make jetty listen on all interfaces
-sed  -i "s/\(-Dfile.encoding=UTF-8\)/\1\n-Djetty.host=0.0.0.0/" /etc/default/rudder-jetty
+grep -q "-Djetty.host=0.0.0.0" /opt/rudder/etc/rudder-jetty.conf || echo 'JAVA_OPTIONS="-Djetty.host=0.0.0.0"' >> /opt/rudder/etc/rudder-jetty.conf
```
Comment (jooooooon) [inline scripts/rudder-multiserver-setup/rudder-ldap.sh]: I doubt we want to overwrite JAVA_OPTIONS completely. Surely you want to add to it's contents?

Comment (Kegeruneku) [inline scripts/rudder-multiserver-setup/rudder-ldap.sh]: Well, the defaults options are in /etc/default/rudder-jetty, I'm only adding entries here that will be added to the default ones in /etc/default/rudder-jetty (that sources /opt/rudder/etc/rudder-jetty.conf )

**Comment by Kegeruneku** [comment]
This has nothing to do on master :)

**Comment by peckpeck** [comment]
OK it matches the bug description.
We will now have this setting in /opt/rudder/etc/rudder-jetty.conf instead

**Pull request merged**